code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -5362,6 +5362,13 @@
 					H.adjustToxLoss(3)
 				H.adjustToxLoss(0.3)
 
+/datum/reagent/ethanol/deadrumm/pintpointer
+	name = "Pintpointer"
+	id = PINTPOINTER
+	description = "An attempt to create a navigation system which even a drunk spaceman can use."
```

Comment:
This description that is only ever visible when analyzed at a chemmaster is better than the description of the item itself